services/fetchUserRepos.ts

```diff
@@ -0,0 +1,15 @@
+import axios from 'axios'
+import { headers } from './commons/getHeaders'
+
+export default async (reposUrl: string): Promise<Array<string>> => {
+	const { data: items } = await axios.get(reposUrl, { headers: headers() })
```

Comment:
I can see us being rate-limited by doing this, don't you think?

Reply:
I think you want to query languages in the database so I tried to get all programming languages in each repo. That was a easiest way using only the language returned in Repo URL. What would be the best approach here?